tests/test_stateful_browser.py

```diff
@@ -367,11 +372,12 @@ def test_select_form_tag_object():
 
 def test_referer_follow_link(httpbin):
     browser = mechanicalsoup.StatefulBrowser()
-    browser.open(httpbin.url)
+    open_legacy_httpbin(browser, httpbin)
+    start_url = browser.get_url()
```

Comment:
If we want to, we can change this test to use `httpbin + "/links"` instead of the home page to minimize the interference of the change to httpbin.

Reply:
Good to know, but I guess I'm too lazy to change it and the current code works well ;-).